src/components/Home/PipelineSection/PipelineSection.tsx

```diff
@@ -140,8 +141,15 @@ export const PipelineSection = withSuspenseWrapper(() => {
 
   if (pipelines.size === 0) {
     return (
-      <BlockStack gap="2" align="center" className="mt-4">
-        <Paragraph>No pipelines found.</Paragraph>
+      <BlockStack gap="2" align="center">
+        <Paragraph size="md" tone="subdued">
+          Seems that you don&apos;t have any pipelines yet. Get started quickly
+          with our pre-built pipeline templates. Each template demonstrates
+          different ML workflow patterns and best practices. Simply click on any
+          template to import it and start customizing.
```

Comment:
imo this copy text can be greatly simplified

```suggestion
          You don&apos;t have any pipelines yet. Get started with a template below.
```

Reply:
A little touch up to the spacing and layout and I think it looks good

![image.png](https://app.graphite.dev/user-attachments/assets/ceef0f7c-d2d1-4544-af09-dca59b4ff643.png)